components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -18,45 +16,28 @@ export default class ConfirmIntegration extends React.Component {
             team: PropTypes.object,
             location: PropTypes.object,
             commands: PropTypes.object,
-            loading: PropTypes.bool,
+            oauthApps: PropTypes.object,
+            outgoingHooks: PropTypes.object,
         };
     }
 
     constructor(props) {
         super(props);
-
-        this.handleIntegrationChange = this.handleIntegrationChange.bind(this);
         this.handleKeyPress = this.handleKeyPress.bind(this);
```

Comment:
You may remove this binding. Below, you may change to arrow function, like so:
```
handleKeyPress = (e) => {
    ...
}
```

Reply:
Was thinking about this but my latest javascript readings confused me. Updated